research/src/pages/popup/popup.research.explainer.mdx

```diff
@@ -342,80 +479,49 @@ This section contains several HTML examples, showing how various UI elements mig
-
-* The full screen API (element.requestFullScreen()).
-* The `<dialog>` element, shown modally (dialog.showModal()).
-* Elements using the prototype `<popup>` element implementation in Chromium.
-
-This table documents what **currently happens** in the Chromium rendering engine, which is the only one to support any top-layer elements besides fullscreen. In particular, it documents the current `<popup>` element prototype, which is just one possibility for the general “popup” API being described here.
-
-**<span style="text-decoration:underline;">Chromium 99.x behavior</span>**
-
-
-<table>
-  <tr><td></td><td colspan="4" >Second Element</td></tr>
-  <tr><td rowspan="4" > First Element</td><td></td><td>Full screen</td><td>Modal dialog</td><td>`&lt;popup>` element</td></tr>
-  <tr><td>Full screen</td><td>The second fullscreen element kicks the first one out of the top layer. ESC closes the second fullscreen element.</td><td>Full screen stays visible, modal is displayed above fullscreen. ESC closes fullscreen <strong>first</strong>, second ESC closes dialog (<strong>backwards</strong>).</td><td>Full screen stays visible, popup is displayed above fullscreen. ESC closes fullscreen <strong>first</strong>, second ESC closes popup (<strong>backwards</strong>).</td></tr>
-  <tr><td>Modal dialog</td><td>Full screen is displayed on top of modal, both are in the top layer. ESC closes full screen, second ESC closes dialog.</td><td>Both dialogs are shown, first under second. First ESC closes second dialog, then first.</td><td>Both are shown, dialog is under popup. Hitting ESC once closes <strong>both</strong> of them.</td></tr>
-  <tr><td>`&lt;popup>` element</td><td>Full screen is displayed on top of popup, both are in the top layer. ESC closes full screen, second ESC closes popup.</td><td>The dialog opening dismisses the popup. Hitting ESC closes the dialog.</td><td>The second popup dismisses the first (assuming they're not “ancestors” via DOM tree or anchor/popup attributes). Hitting ESC closes the popup.</td></tr>
-</table>
+## Exceeding the Frame Bounds
 
+Allowing a popup/top-layer element to exceed the bounds of its containing frame poses a serious security risk: such an element could spoof browser UI or containing-page content. While the [original `<popup>` proposal](https://open-ui.org/components/popup.research.explainer#goals) did not discuss this issue, the [`<selectmenu>` proposal](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/ControlUICustomization/explainer.md#security) does have a specific section at least mentioning this issue. Some top-layer APIs (e.g. the fullscreen API) make it possible for an element to exceed the frame bounds in some cases, great care must be taken in these cases to ensure user safety. Given the complete flexibility offered by the Popup API (any element, arbitrary content, etc.), there would be no way to ensure the safety of this feature if it were allowed to exceed frame bounds.
```

Comment:
Unless I've made a silly mistake, won't the following: 

`While the [original `<popup>` proposal](https://open-ui.org/components/popup.research.explainer#goals)...`

just link to the current page's goals heading? 

Is there a different document that this link should point to, or is the wording just a little ambiguous?

I'd figure that the _original <popup> proposal_ would be an older spec (maybe the abandoned one?)

Reply:
Great catch! That's from before I moved the explainer to here.